### ISSUE TITLE
Kousa: Deploy: is Kousa.Release neecessary?

### ISSUE THREAD

**Issue by ityonemo** [benawad/dogehouse#888]
Is there some reason why the normal Ecto migration mix tasks can't be used instead?

**Comment by ityonemo** [comment]
@benawad

**Comment by praveenperera** [comment]
Yes because mix is a build too which is not available in releases: https://hexdocs.pm/phoenix/releases.html#ecto-migrations-and-custom-commands

**Comment by ityonemo** [comment]
nope, I mean calling the modules directly instead of via an intermediary, those modules should still exist in release.

**Comment by A-Yamout** [comment]
@ityonemo If you are done with this could you close this issue?

**Comment by dark-elixir-shibe** [comment]
not done
